shell/ci/release/docker.sh

```diff
@@ -129,6 +129,26 @@ build_and_push_image() {
     fi
     platformArgumentString+="$platform"
   done
+  args+=("--platform" "$platformArgumentString")
+
+  # tags are the tags to apply to the image. If we're on a git tag,
+  # we'll tag the image with that tag and latest. Otherwise, we'll just
+  # build a latest image for the name "$image" (the name of the image as
+  # shown in the manifest) instead.
+  tags=()
+  if [[ -n $CIRCLE_TAG ]]; then
+    tags+=("$remote_image_name:$CIRCLE_TAG" "$remote_image_name:latest")
+  else
+    tags+=("$image")
+  fi
+  for tag in "${tags[@]}"; do
+    args+=("--tag" "$tag")
+  done
+
+  # When on a tag, we should push the image to the registry.
```

Comment:
Put this & body in `if` on line 139?

Reply:
I didn't like mixing the tags logic... but yeah ig it makes sense.